### PR TITLE
fix: correct resolve image url in js

### PR DESCRIPTION
**Pull request by webdiscus**
Hello @jguneratne,

there is fixed working version.

## Start developing in browser

```
npm start
```
open in your browser the URL: `http://localhost:8080/`

To allow the processing of images via Webpack defined in JS, use the `require('path/to/image.svg')` function.
Webpack detects required images, resolves and generate output image URL.

### CODE REVIEW

File: src/js/display-forecasts.js [hunk 1]
```diff
@@ -15,15 +15,22 @@ export function displayToday(todayData) {
     todayDate.textContent = todayOnScreen;
   }
 
-  weatherIconsData.forEach((icon) => {
```
Comment (webdiscus) [inline src/js/display-forecasts.js:18]: you can export icon data as "iconName" : "imageURL" and access to image by iconName.
You should't iterate an array to find an icon.

File: src/js/weather-icons.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (webdiscus) [inline src/js/weather-icons.json:1]: sorry, delete this file...

File: src/styles/style.css [hunk 3]
```diff
@@ -167,3 +167,7 @@ input {
 .github-logo {
   max-width: 2em;
 }
+
```
Comment (webdiscus) [inline src/styles/style.css:170]: define the icon with, otherwise will be render the image with 0x0 px.

File: src/views/index.html [hunk 4]
```diff
@@ -221,7 +222,7 @@ <h1>Weather App</h1>
               rel="noopener noreferrer"
             >
               <img
-                src="../assets/imgs/icons/github-mark.svg"
+                src="@images/icons/github-mark.svg"
```
Comment (webdiscus) [inline src/views/index.html:225]: use webpack alias instead relative path

**Comment by webdiscus** [comment]
if the solution works for you, can you please accept my answer on stackoverflow.

**Comment by jguneratne** [comment]
Thank you so much, I definitely learned something today!

**Pull request merged**